cmd/lakectl/cmd/local_commit.go

```diff
@@ -52,6 +52,14 @@ var localCommitCmd = &cobra.Command{
 			DieErr(err)
 		}
 
+		if idx.ActiveOperation != "" && idx.ActiveOperation != string(commitOperation) {
```

Comment:
We don't really need to differentiate between commit operation and the rest, I think we should provide this confirmation regardless:
```suggestion
		if idx.ActiveOperation != "" {
```